packages/fxa-auth-server/lib/error.js

```diff
@@ -785,6 +786,15 @@ AppError.unverifiedPrimaryEmailNewlyCreated = () => {
   });
 };
 
+AppError.maxSecondaryEmailsReached = () => {
+  return new AppError({
+    code: 400,
+    error: 'Bad Request',
+    errno: ERRNO.MAX_SECONDARY_EMAILS_REACHED,
+    message: 'You cannot have more than 10 secondary emails',
```

Comment:
From the comments, I didn't see where the limit of 10 came from. That seems a bit high to me. I'm in favor of keeping this number fairly low, probably 3 or less. This could easily be increased after we gather stats about usage.

Reply:
Fair enough. 10 was an arbitrary choice. @davismtl you cool with 3?